### ISSUE TITLE
sp_BlitzWho - add lead blocker if they're sleeping

### ISSUE THREAD

**Issue by BrentOzar** [BrentOzarULTD/SQL-Server-First-Responder-Kit#820]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
Begin a transaction in one window, and lock a table, but leave the transaction open. He's sleeping.

In another window, start a transaction that will get blocked by the first one.

If you run sp_BlitzWho, it doesn't show sleeping sessions, so the lead blocker doesn't show up in the list.

**What is the expected behavior?**
It'd be cool if we showed sleeping sessions if they have open transactions or blocks.

**Comment by hellomahesh** [comment]
Is this resolved? Because I faced this issue today with latest version of sp_BlitzWho from Github.

**Comment by BrentOzar** [comment]
Yes. If you experience it, can you post a new issue with repro steps? Thanks!

---
Tiny glass keyboard
Typos flowing like rivers
As winter snow thaws
